doc/guide/admin.rst

```diff
@@ -852,6 +852,272 @@ to be added to one or more PAM stacks, e.g.
 
   account  sufficient pam_flux.so
 
+********************
+Pre-flight Checklist
+********************
+
+Here are some things to check before going live with a new Flux system
+instance.
+
+Do I have all the right packages installed?
+===========================================
+
+.. list-table::
+   :header-rows: 1
+
+   * - Package name
+     - Where
+
+   * - flux-core
+     - All nodes
+
+   * - flux-security
+     - All nodes
+
+   * - flux-sched
+     - All nodes
+
+   * - flux-pam (optional)
+     - All nodes
+
+   * - flux-accounting (optional)
+     - Leader (management) node only
```

Comment:
Isn't the scheduler only required on the leader node (modules are only loaded on the leader), and flux-accounting should be installed on all nodes so that users/admins have access to the query tools like `flux account` (which can be run on any node and query the rank 0 accounting service) The accounting systemd service should only be enabled on the management node though.

Reply:
> Isn't the scheduler only required on the leader node

For the system instance, but presumably the system instance launches batch jobs with schedulers on other nodes

> flux-accounting should be installed on all nodes so that users/admins have access to the query tools like flux account (which can be run on any node and query the rank 0 accounting service) The accounting systemd service should only be enabled on the management node though.

Oh good point. I hadn't thought about that.  Well @cmoussa1 might want to weigh in here.  The accounting guide says management node only but that may predate the work to let the clients work remotely.

https://flux-framework.readthedocs.io/en/latest/guides/accounting-guide.html#installing-software-packages

Isn't the systemd unit enabled by default though?

Reply:
> For the system instance, but presumably the system instance launches batch jobs with schedulers on other nodes

Oh heh, thinking brain not thinking.

Reply:
> Isn't the systemd unit enabled by default though?

It seems to be disabled by default (at least in the RPM?) I kind of forget how this works right now, but here's the output from a TOSS login node:
```
● flux-accounting.service - Flux accounting service
   Loaded: loaded (/usr/lib/systemd/system/flux-accounting.service; disabled; vendor preset: disabled)
   Active: inactive (dead)
```

I think `vendor preset: disabled` means the service needs to be explicitly enabled with `systemctl enable flux-accounting`

Reply:
I think the accounting guide needs to be updated with some notes regarding the new accounting systemd service that was added a little while back (there is currently an open PR in flux-framework/flux-docs#216 that looks to clarify this). It used to be the case where you could only run flux-accounting commands on the management node, but that is no longer the case, you should be able to run it from anywhere on the cluster.

My understanding was that flux-accounting is typically only installed on the management node, however, since that is where the DB is installed.

Reply:
> My understanding was that flux-accounting is typically only installed on the management node, however, since that is where the DB is installed.

This is not the case on our TOSS clusters running flux. flux-accounting is installed at least on the login nodes as well.

Reply:
My mistake then! Sorry for the confusion.

Reply:
Shall we just simplify and have all packages installed on all nodes?

Reply:
Sure!  Thanks @cmoussa1 for the follow up.